addons/website_sale/__manifest__.py

```diff
@@ -168,6 +168,9 @@
         'website.backend_assets_all_wysiwyg': [
             'website_sale/static/src/js/components/wysiwyg_adapter/wysiwyg_adapter.js',
         ],
+        'portal.assets_chatter': [
+            'website_sale/static/src/chatter/frontend/**/*',
```

Comment:
Having `frontend` as "bundle" name in the path is confusing because the files are not in the `frontend` bundle but in the "portal chatter" bundle.